src/main/java/htsjdk/tribble/util/ParsingUtils.java

```diff
@@ -454,6 +461,7 @@ private static URLHelper getURLHelper(Class helperClass, URL url) {
      * The default helper class is {@link RemoteURLHelper}, which delegates to FTP/HTTP
      * helpers as appropriate.
      *
+     * @deprecated use {@code registerHelperFactory}
```

Comment:
If you use an `@link` or `@see` tag instead the javadoc and IDE can provide links which is nice.